apps/ern-ithaca/vite.config.js

```diff
@@ -0,0 +1,53 @@
+import { defineConfig } from "vite";
+import vue from "@vitejs/plugin-vue";
+
+const host = "https://ithaca-emx2test.molgeniscloud.org/";

```

Comment:
Something to discuss during a dev-meet I guess, but is it ok to let apps refer to test servers like this. I'd prefer to have it run on localhost so I can test things in isolation.